ftplugin/copilot_chat.vim

```diff
@@ -4,7 +4,7 @@ if exists('g:copilot_chat_disable_mappings') && g:copilot_chat_disable_mappings
   finish
 endif
 
-nnoremap <buffer> <leader>cs :CopilotSubmit<CR>
-nnoremap <buffer> <CR> :CopilotSubmit<CR>
+nnoremap <buffer> <leader>cs :CopilotChatSubmit<CR>
+nnoremap <buffer> <CR> :CopilotChatSubmit<CR>
```

Comment:
I strongly feel the `<cr>` map should stay.  It's a sensible mapping to only exist in `copilot_chat` buffers.  If you wanted to be pragmatic you could first check if it exists before mapping it.

Reply:
I'm in the same boat that this is the only real mapping I care about / think is worth enforcing for purely QOL purposes

Reply:
No, I just can't read diffs, apparently 🙃  Sorry 'bout that.